pkg/sdkserver/localsdk.go

```diff
@@ -443,42 +456,44 @@ func (l *LocalSDKServer) Close() {
 	l.compare()
 }
 
-// EqualSets tells whether a and b contain the same elements.
+// EqualSets tells whether expected and received slices contain the same elements.
 // A nil argument is equivalent to an empty slice.
-func EqualSets(a, b []string) bool {
+func (l *LocalSDKServer) EqualSets(expected, received []string) bool {
 	aSet := make(map[string]bool)
 	bSet := make(map[string]bool)
-	for _, v := range a {
+	for _, v := range expected {
 		aSet[v] = true
 	}
-	for _, v := range b {
+	for _, v := range received {
 		if _, ok := aSet[v]; !ok {
+			l.logger.Infof("Found an element which was not expected %s", v)
```

Comment:
.WithField().Error() ?

Reply:
Thanks, here definitely should go an error, as well as this helps me to identify one `fmt.Print` usage.